src/event/event_config.h

```diff
@@ -76,16 +76,19 @@
 #if DISPATCH_EVENT_BACKEND_KEVENT
 #	if defined(EV_SET_QOS)
 #		define DISPATCH_USE_KEVENT_QOS 1
-#		ifndef KEVENT_FLAG_IMMEDIATE
-#		define KEVENT_FLAG_IMMEDIATE 0x001
-#		endif
-#		ifndef KEVENT_FLAG_ERROR_EVENTS
-#		define KEVENT_FLAG_ERROR_EVENTS 0x002
-#		endif
 #	else
 #		define DISPATCH_USE_KEVENT_QOS 0
 #	endif
 
+/// The flags here are set unconditionally (from DISPATCH_USE_KEVENT_QOS).
```

Comment:
This also looks like an odd change - why has the indentation changed?

Reply:
These flags were being used everywhere and unconditionally. Originally they were only defined if `defined(EV_SET_QOS)`, which is not true on FreeBSD. The indentation didn't change per se, it was moved one level down and indentation corrected accordingly.

 - In the epoll `KEVENT_FLAG_IMMEDIATE` is emulated with a timeout, the same must be done on kevent (this is not done yet -- this PR is only to get the library building and submitting things in small batches)
 - `KEVENT_FLAG_ERROR_EVENTS` already has a emulation (line 611)